KomaMRIBase/src/motion/actions/simpleactions/Rotate.jl

```diff
@@ -1,5 +1,9 @@
+struct CenterOfMass end
+Base.:(≈)(::CenterOfMass, ::CenterOfMass) = true
+Base.:(≈)(a, b) = (a isa CenterOfMass || b isa CenterOfMass) ? false : Base.≈(a, b)
```

Comment:
This is still type piracy. You are redefining `Base.:(≈)(a, b)`.

You probably want these:
```julia
Base.:(≈)(a::CenterOfMass, b)
Base.:(≈)(a, b::CenterOfMass)
```